pkgs/by-name/va/vala-panel-appmenu/package.nix

```
@@ -0,0 +1,137 @@
+# to prevent bloating this package, please choose what integration you want
+# to use manually, like this:
+#   vala-panel-appmenu.override { enableMate = true; }
+# the package compiles very quickly, so there is no sense to have every
+# single combination to be build by hydra
+{
+  lib,
+  budgie-desktop,
+  fetchFromGitLab,
+  glib,
+  gobject-introspection,
+  gtk3,
+  jdk,
+  libdbusmenu,
+  libwnck,
+  libxkbcommon,
+  mate,
+  meson,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  stdenv,
+  stripJavaArchivesHook,
+  vala,
+  vala-panel-appmenu,
+  wrapGAppsHook3,
+  xfce,
+
+  enableXfce ? false,
+  enableBudgie ? false,
+  enableValapanel ? false, # NOTE: requires https://github.com/rilian-la-te/vala-panel which is not packaged
+  enableMate ? false,
+  enableJayatana ? false,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vala-panel-appmenu";
+  version = "25.04";
+
+  src = fetchFromGitLab {
+    owner = "vala-panel-project";
+    repo = "vala-panel-appmenu";
+    tag = finalAttrs.version;
+    hash = "sha256-v5J3nwViNiSKRPdJr+lhNUdKaPG82fShPDlnmix5tlY=";
+    fetchSubmodules = true;
```

```suggestion
    fetchSubmodules = true;
    hash = "sha256-v5J3nwViNiSKRPdJr+lhNUdKaPG82fShPDlnmix5tlY=";
```
Nit: It's better to have `fetchSubmodules = true` above the hash as it affects it.

---

Bu there are no submodules?